README.md

```diff
@@ -1,22 +1,19 @@
 homebrew-font
 =============
 
-# Install fonts
+# Installing Ricty fonts
 
-Compile Ricty Font
-
-* Install
-   * Recommend
+Recommended installation by tapping the repository
 
 ```
-$ brew tap sanemat/font
+$ brew tap sanemat/homebrew-font
```

Comment:
Okay, I revised the commit to leave it as `brew tap sanemat/font`. It looks like the pull request was updated automatically.